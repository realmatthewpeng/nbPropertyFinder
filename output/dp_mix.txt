Total number of lines found: 21

w[:, 0] = beta[:, 0]
w[:, 0] = beta[:, 0]
w[:, 0] = beta[:, 0]
w[:, 0] = beta[:, 0]
old_faithful_df = get_rdataset('faithful', cache=True).data[['waiting']]
old_faithful_df['std_waiting'] = (old_faithful_df.waiting - old_faithful_df.waiting.mean()) / old_faithful_df.waiting.std()
old_faithful_df.head()
N = old_faithful_df.shape[0]
ax.bar(plot_w - 0.5, trace['w'].mean(axis=0), width=1., lw=0);
ax.plot(x_plot, post_pdfs.mean(axis=0),
ax.plot(x_plot, (trace['w'][:, np.newaxis, :] * post_pdf_contribs).mean(axis=0)[:, 0],
ax.plot(x_plot, post_pdfs.mean(axis=0),
ax.plot(x_plot, (trace['w'][:, np.newaxis, :] * post_pdf_contribs).mean(axis=0),
sunspot_df.head()
N = sunspot_df.shape[0]
trace = trace_[50000::50]
ax.bar(plot_w - 0.5, trace['w'].mean(axis=0), width=1., lw=0);
ax.plot(x_plot, post_pmfs.mean(axis=0),
ax.plot(x_plot, (trace['w'][:, np.newaxis, :] * post_pmf_contribs).mean(axis=0),
ax.plot(x_plot, post_pmfs.mean(axis=0),
ax.plot(x_plot, (trace['w'][:, np.newaxis, :] * post_pmf_contribs).mean(axis=0)[:, 0],

Incorrect Lines:

w[:, 0] = beta[:, 0]
w[:, 0] = beta[:, 0]
w[:, 0] = beta[:, 0]
w[:, 0] = beta[:, 0]
old_faithful_df = get_rdataset('faithful', cache=True).data[['waiting']]
N = old_faithful_df.shape[0]
ax.bar(plot_w - 0.5, trace['w'].mean(axis=0), width=1., lw=0);
ax.plot(x_plot, post_pdfs.mean(axis=0),
ax.plot(x_plot, (trace['w'][:, np.newaxis, :] * post_pdf_contribs).mean(axis=0)[:, 0],
ax.plot(x_plot, post_pdfs.mean(axis=0),
ax.plot(x_plot, (trace['w'][:, np.newaxis, :] * post_pdf_contribs).mean(axis=0),
trace = trace_[50000::50]
ax.bar(plot_w - 0.5, trace['w'].mean(axis=0), width=1., lw=0);
ax.plot(x_plot, post_pmfs.mean(axis=0),
ax.plot(x_plot, (trace['w'][:, np.newaxis, :] * post_pmf_contribs).mean(axis=0),
ax.plot(x_plot, post_pmfs.mean(axis=0),
ax.plot(x_plot, (trace['w'][:, np.newaxis, :] * post_pmf_contribs).mean(axis=0)[:, 0],

Precision = 0.19047619047619047
Recall = 1.0

