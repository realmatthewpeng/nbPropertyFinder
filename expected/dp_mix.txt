old_faithful_df['std_waiting'] = (old_faithful_df.waiting - old_faithful_df.waiting.mean()) / old_faithful_df.waiting.std()
old_faithful_df.head()
sunspot_df.head()
N = sunspot_df.shape[0]